app/interactors/update_user.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateUser
+  include Interactor
+
+  def call
+    user = context.current_user.update(context.permitted_params)
+
+    context.fail!(errors: user.errors) unless user
```

Comment:
Could you please add tests to this interactor? I'm not sure that it works as expected.